src/main/java/picard/analysis/AlignmentSummaryMetricsCollector.java

```diff
@@ -261,7 +261,7 @@ public void acceptRecord(final SAMRecordAndReference samRecordAndReference) {
         @Override
         public void finish() {
             //summarize read data
-            if (metrics.TOTAL_READS > 0) {
+            if (metrics.PF_READS > 0) {
```

Comment:
I think it might be a good idea to add an `else` statement here, that prints a warning if the number of `PF_READS` is 0, just in case someone actually reads the logs and is wondering why their data looks weird. I think something that explains the possibilities - either lack of reads or they were all QC'd out - might help users debugging. There are a number of other tools that use a logger to print warnings you could use as a template. (Though I'm not sure what the standard is for "where" loggers get invoked. Maybe down inside a metrics collector is too low level? I'd check the tool that calls this to see what it does, if anything.)